modules/ROOT/pages/_partials/sync/adding_keycloak.adoc

```diff
@@ -0,0 +1,57 @@
+= Prerequisites
+
+- There is a Keycloak service available.
+- You have provisioned a sync application using our playbook.
+- oc tools must be installed
+- You must first create a client for your application in the Keycloak Administration Console, and then click on the `Installation` tab, select `Keycloak OIDC JSON` for `Format` Option, and then click on `Download`. Save the downloaded `keycloak.json`.
+
+= Connecting your application to Keycloak
+
+Any application that connects to keycloak must consume a keycloak.json file.  This guide demonstrates how to add a keycloak.json file to your sync application deployment.  It will create the environment variable `KEYCLOAK_CONFIG` that contain the path to keycloak.json.  It is still your application's responsibility to consume this file.  We have provided an link:https://github.com/aerogear/voyager-server/tree/master/examples/keycloak[example project].
+
+= Create a Keycloak Secret
+
+The following command will create a secret called `sync-keycloak-doc`.  It will contain one key, `keycloak` and a value, the text of your keycloak.json file.
+
+```bash
```

Comment:
Hey @secondsun this is actually not valid asciidoc syntax.

It should be something like this:

```asciidoc
[source, bash]
----
echo "hello world"
----
```